superdesk/data_updates/00003_20170814-114652_audit.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Author  : superdesk
+# Creation: 2017-08-14 11:47
+
+from superdesk.commands.data_updates import DataUpdate
+from superdesk import get_resource_service
+from eve.utils import config
+
+
+class DataUpdate(DataUpdate):
+
+    resource = 'audit'
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        for audit in mongodb_collection.find({}):
+            audit_id = get_resource_service(self.resource)._extract_doc_id(audit.get('extra'))
+            print(mongodb_collection.update({'_id': audit.get(config.ID_FIELD)},
```

Comment:
@petrjasek Yes we've been tuning it on our UAT system, It's just a painfully slow process! 

Do you think the index should be coded in the resource. or just create it in Mongo externally?